.expeditor/release.omnibus.yml

```diff
@@ -55,8 +55,8 @@ builder-to-testers-map:
     - sles-15-x86_64
   sles-15-aarch64:
     - sles-15-aarch64
-  solaris2-5.11-i386:
-    - solaris2-5.11-i386
+  # solaris2-5.11-i386:
+  #   - solaris2-5.11-i386
```

Comment:
can we remove the commented code? We will still know what was removed from this commit if anything is needed in future